build/uno.winui.common.targets

```diff
@@ -194,4 +194,22 @@
 					 Condition="'$(_CurrentTrimmedAndroidSDLVersion)' &lt; '$(UnoUIMinAndroidSDKVersion)'" />
 	</Target>
 
+	<!-- Validation for https://github.com/unoplatform/uno/issues/9430 and the static registrar requirement -->
+	<Target Name="_ValidateIOSStaticRegistrar"
+			BeforeTargets="BeforeBuild"
+			Condition="
+			'$(UnoDisableValidateIOSStaticRegistrar)'==''
+			and (
+				('$(TargetFramework)'=='net6.0-ios' and '$(RuntimeIdentifier)'=='iossimulator-x64')
```

Comment:
In theory this would apply to `iossimulator-arm64` too but it is currently broken
https://github.com/xamarin/xamarin-macios/issues/15666

and hopefully this workaround won't be needed soon (once xcode14 update is out) so it's likely not important to add here